compiler/common-artifacts/src/TestDataGenerator.cpp

```diff
@@ -120,6 +151,11 @@ int entry(int argc, char **argv)
     .required(false)
     .nargs(0)
     .help("Put a fixed seed into the random number generator");
+  arser.add_argument("--input_range")
+    .required(false)
+    .nargs(3)
+    .type(arser::DataType::STR_VEC)
+    .help("Set random number range for the input as 'name min max'");
```

Comment:
This could be a bit more helpful. Whether it is closed interval or open one.
```suggestion
    .help("Set random number range [min max] for the input as 'name min max'");
```